src/test/kotlin/com/terraformation/backend/customer/db/ProjectStoreTest.kt

```diff
@@ -420,4 +421,40 @@ class ProjectStoreTest : DatabaseTest(), RunsAsUser {
       eventPublisher.assertEventPublished(ProjectDeletionStartedEvent(projectId))
     }
   }
+
+  @Nested
+  inner class FetchCohortData {
+    @Test
+    fun `fetches project's cohort data`() {
+      val cohortId = insertCohort()
+      val participantId = insertParticipant(cohortId = cohortId)
+      val projectId1 = insertProject(participantId = participantId)
+
+      val expected =
+          ProjectCohortData(cohortId = cohortId, cohortPhase = CohortPhase.Phase0DueDiligence)
+      val actual = store.fetchCohortData(projectId1)
+
+      assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `returns null if no permission to read cohort`() {
+      val cohortId = insertCohort()
+      val participantId = insertParticipant(cohortId = cohortId)
+      val projectId1 = insertProject(participantId = participantId)
+
+      every { user.canReadCohort(any()) } returns false
+
+      val actual = store.fetchCohortData(projectId1)
+      assertEquals(null, actual)
+    }
+
+    @Test
+    fun `returns null if the project is not associated to a cohort`() {
+      every { user.canReadCohort(any()) } returns false
```

Comment:
This test name seems slightly inaccurate:

- It's testing a project with no participant, rather than one with a participant that has no cohort. (Could test both of those cases.)
- The permission should probably be true, not false, so that lack of permission isn't part of what's being tested here. Permissions are covered in the previous case.

Reply:
I removed that permission check, updated the name of the test, and added another test